src/Service.Tests/Snapshots/ConfigurationTests.TestReadingRuntimeConfigForCosmos.verified.txt

```diff
@@ -282,42 +298,32 @@
             Role: field-mutation-with-read-permission,
             Actions: [
               {
-                Action: Update,
+                Action: Read
+              }
+            ]
+          },
+          {
+            Role: anonymous,
+            Actions: [
+              {
+                Action: Read,
                 Fields: {
                   Exclude: [
-                    name
+                    *
                   ],
                   Include: [
-                    id,
-                    type
+                    *
                   ]
                 }
```

Comment:
where is this used? is this a test validating that `Exclude: ["*"]` takes precedence over included fields?

Reply:
this permission was not getting used. Hence removed it https://github.com/Azure/data-api-builder/blob/072c910133d6b86360072aaf3554bfade42da4ac/src/Service.Tests/Snapshots/ConfigurationTests.TestReadingRuntimeConfigForCosmos.verified.txt#L199